composer.json

```diff
@@ -25,15 +25,15 @@
     ],
     "require": {
         "php": "^7.2|^8.0.2|^8.1",
```

Comment:
Laravel 11 also requires PHP 8.2 😃
Consider appending `|^8.2` to these versions, or replace both version 8's with `^8` to support any PHP 8 installation.